docs-source/content/userguide/managing-domains/model-in-image/runtime-updates.md

```diff
@@ -0,0 +1,325 @@
+
+ - You can add or alter an MBean attribute by specifying a YAML snippet along with its parent bean hierarchy that references an existing MBean and the attribute. For example, to add or alter the max capacity of a data source named `mynewdatasource`:
+
+  ```
+  resources:
+    JDBCSystemResource:
+      mynewdatasource:
+        JdbcResource:
+          JDBCConnectionPoolParams:
+            MaxCapacity: 5
+  ```
+
+  For more information, see [Using Multiple Models](https://github.com/oracle/weblogic-deploy-tooling#using-multiple-models) in the WebLogic Deploy Tooling documentation.
+
+ - There is no way to directly delete an attribute from an MBean that's already been specified by a model file. The work-around is to do this using two model files: (a) add a model file that deletes the named bean/resource that is a parent to the attribute you want to delete, and (b) add another  subsequent model file that fully defines the named bean/resource but without the attribute you want to delete.
+
+ - The following runtime updates haven't been tested and are _not_ supported in the first release of Model in Image. If you need to make these kinds of updates, consider shutting down your domain entirely before making the change:
+   - Adding, removing, or altering the network configuration of an existing Managed Server. This includes, but isn't limited to network channels, ports, and cluster addresses.
+   - Adding a Managed Server to an existing configured cluster.
+   - Altering the `cluster-size` or `max-cluster-size` of an existing dynamic cluster.
+   - TBD This needs some research. Check with QA, etc.
+
+#### Changing a domain resource 'restartVersion'
+
+As was mentioned in the [overview](#overview), one way to tell the operator to apply your configuration changes to a running domain is by altering the domain resource `restartVersion`. Here are some common ways to do this:
+
+ - You can alter `restartVersion` interactively using `kubectl edit -n MY_NAMESPACE domain MY_DOMAINUID`.
+
+ - If you have your domain's resource file, then you can alter this file and call `kubectl apply -f` on the file.
+
+ - You can use the Kubernetes `get` and `patch` commands. Here's a sample automation script:
+
+   ```
+   #!/bin/bash
+   NAMESPACE=${1:-sample-domain1-ns}
+   DOMAINUID=${2:-sample-domain1}
+   currentRV=$(kubectl -n ${NAMESPACE} get domain ${DOMAINUID} -o=jsonpath='{.spec.restartVersion}')
+   if [ $? = 0 ]; then
+     # we enter here only if the previous command succeeded
+
+     nextRV=$((currentRV + 1))
+
+     echo "@@ Info: Rolling domain '${DOMAINUID}' in namespace '${NAMESPACE}' from restartVersion='${currentRV}' to restartVersion='${nextRV}'."
+
+     kubectl -n ${NAMESPACE} patch domain ${DOMAINUID} --type='json' \
+       -p='[{"op": "replace", "path": "/spec/restartVersion", "value": "'${nextRV}'" }]'
+   fi
+   ```
+
+#### Using the WDT Discover Domain Tool
```

Comment:
This is a strange way of using WDT discover, why would you need to discover a domain running in the pod?  Most users will version control their models and domain.yamls.  To make updates to a running domain you can use a WDT sparse model as the following sample shows.  
If the intent of this sample is to compare old and new models or recover a lost model then that is what the description should say.

Reply:
Okay, now I understand your earlier comment, @mriccell.  I thought the use of WDT discover was going to be to help generate the initial domain yaml.

Reply:
Tom AI:  Put bash/ephemeral part into P4 for internal use only.   Only talk about domain-home without saying where it is.

Reply:
Done.